shared/src/packages/orders/libs/enums/orders-validation-message.enum.ts

```diff
@@ -0,0 +1,9 @@
+const OrdersValidationMessage = {
+  // COMPANY_NAME_REQUIRED: 'Company name is required',
+  // TAX_NUMBER_REQUIRED: 'Tax number is required',
```

Comment:
Will we use it later? If not - then remove commented code

Reply:
removed